src/WebJobs.Script/OutOfProc/Rpc/RpcWorkerContext.cs

```diff
@@ -12,8 +12,16 @@ internal class RpcWorkerContext : WorkerContext
 
         public int MaxMessageLength { get; set; }
 
-        public override string GetFormatedArguments()
+        public override string GetFormattedArguments()
         {
+            if (ServerUri == null)
```

Comment:
These are not arguments. At least changing this to an `InvalidOperationException` would make more sense, but it this state is invalid, you may want to consider enforcing this in a constructor.